.thothTemplate.yaml

```diff
@@ -0,0 +1,14 @@
+host: '{THOTH_HOST}'
+tls_verify: false
+requirements_format: pipenv
+
+runtime_environments:
+  - name: '{os_name}:{os_version}'
+    operating_system:
+      name: 'ubi'
```

Comment:
```
name: 'rhel'
version: '8.0'
```

Maybe we need to change this also in other repos. As we use UBI:8, we should be able to directly use the autodiscovery parameters:

```
name: '{os_name}'
version: '{os_version}'
```

Reply:
fair point :+1: